### PR TITLE
Update typetalk.py

### PR DESCRIPTION
**Pull request by is2ei**
Now Typetalk uses "com" domain.

+label: docsite_pr

##### SUMMARY
Typetalk usess "com" domain now.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
typetalk

### CODE REVIEW

**Comment by is2ei** [comment]
@ansibot 
I added!

File: lib/ansible/modules/notification/typetalk.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 version_added: "1.6"
 short_description: Send a message to typetalk
 description:
-  - Send a message to typetalk using typetalk API ( http://developers.typetalk.in/ )
+  - Send a message to typetalk using typetalk API ( http://developers.typetalk.com/ )
```
Comment (mkrizek) [inline lib/ansible/modules/notification/typetalk.py]: `http://developers.typetalk.com/` does NOT work for me. `http://developers.typetalk.in` works fine.

**Comment by is2ei** [comment]
@mkrizek 
Thanks! I fixed the URL to "https://developer.nulab-inc.com/docs/typetalk/"
That's the latest URL.

**Comment by is2ei** [comment]
@mkrizek @dagwieers 
Hi, I fixed the URL

**Pull request closed without merging**